js/jquery.mapael.js

```diff
@@ -290,9 +290,16 @@
                             delete plots[deletedPlots[i]];
                         }
                     }
+                // Delete ALL plots if deletedPlots is set to "all"
+                } else if (deletedPlots === "all") {
+                    $.each(plots, function(id, elem) {
+                        fnRemoveElement(elem);
+                    });
+                    // Empty plots array
+                    plots.length = 0;
```

Comment:
Why haven't you wrote "plots = {};" for resetting the object ? Is your way faster ?